docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+| Features               | RPC | Horizon | Galexie | Hubble |
+| ---------------------- | --- | ------- | ------- | ------ |
+| Real-time Data         | ✅  | ✅      | ✅      | ❌     |
+| Historical Data        | ❌  | ❌      | ✅      | ✅     |
+| Classic Data           | ✅  | ✅      | ✅      | ✅     |
+| Soroban Data           | ✅  | ❌      | ✅      | ✅     |
```

Comment:
Horizon actually does have Soroban data (SAC, soroban type operations). Does it warrant making the chart more "detailed"?

Reply:
updated in 0f5207ad468791a2bbf71c4e86ad11e39da74dd7